gallery/src/styles/Nivo/NivoPage.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode, NxP, NxH3, NxTextLink } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NivoLineExample from './NivoLineExample';
+import NivoBarExample from './NivoBarExample';
+import NivoPieExample from './NivoPieExample';
+
+const NivoLineExampleCode = require('./NivoLineExample?raw');
+const NivoBarExampleCode = require('./NivoBarExample?raw');
+const NivoPieExampleCode = require('./NivoPieExample?raw');
+const NivoLineExampleData = require('./lineData?raw');
+const NivoBarExampleData = require('./barData?raw');
+const NivoPieExampleData = require('./pieData?raw');
+
+const NivoPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxTextLink external href="https://nivo.rocks/components/">Nivo</NxTextLink> charts are
+        recommended for data visualization. The charts should be within an element with a definite
+        height. If they do not, the chart will have a height of zero and will not be visible.
+      </NxP>
+      <NxH3>Accessibility</NxH3>
```

Comment:
This isn't really accessibility info

Reply:
> This isn't really accessibility info

Moved it: d40c66035941748850e5e1a8ed69369590574d45